src/components/views/rooms/RoomList.tsx

```diff
@@ -187,6 +222,10 @@ export default class RoomList extends React.PureComponent<IProps, IState> {
             isNameFiltering: !!RoomListStore.instance.getFirstNameFilterCondition(),
         };
 
+        // shallow-copy from the template as we need to make modifications to it
+        this.tagAesthetics = Object.assign({}, TAG_AESTHETICS);
```

Comment:
we have this sort of shallow copy mildly abstracted these days via `objectFastClone` (or similar naming)

Reply:
ah, I'm a bit surprised we prefer custom stuff over native JS, but OK.